vllm/core/scheduler.py

```diff
@@ -125,13 +125,13 @@ def _schedule(self) -> SchedulerOutputs:
 
         # Join waiting sequences if possible.
         if not self.swapped:
-            ignored_seq_groups: List[SequenceGroup] = []
-            scheduled: List[SequenceGroup] = []
+            ignored_seq_groups: deque[SequenceGroup] = deque()
+            scheduled: deque[SequenceGroup] = deque()
```

Comment:
For `ignored_seq_groups` and `scheduled`, we only append new items to them. Why don't we keep using `List` for them?